plugins/inventory/xen_orchestra.py

```diff
@@ -57,6 +57,14 @@
             description: Use wss when connecting to the Xen Orchestra API
             type: boolean
             default: true
+        strict:
+            version_added: 4.1.0
+        compose:
+            version_added: 4.1.0
+        groups:
+            version_added: 4.1.0
+        keyed_groups:
+            version_added: 4.1.0
```

Comment:
Please add the proper documentation to the new parameters.

Reply:
Hi, I thought [this line](https://github.com/ansible-collections/community.general/blob/a20c7e55562f8faf5823181c691e7275924b619e/plugins/inventory/xen_orchestra.py#L21) would do the trick?

Reply:
Actually this isn't needed, since the plugin has just been added to `main` and `stable-4`. `version_added` and a changelog fragment would only be needed if the plugin would have already been present in an earlier release. But since it first appears in 4.1.0, these extras are not yet needed.

Reply:
(Except of course if this PR misses the 4.1.0 deadline, then both changelog fragment and the `version_added` lines are needed.)